src/target-i386/op_helper.c

```diff
@@ -5966,3 +5983,43 @@ void helper_se_opcode(uint64_t opcode) {
 #endif
 }
 #endif
+
+#if defined(CONFIG_SYMBEX) && !defined(SYMBEX_LLVM_LIB)
+uint8_t RR_cpu_float_status(void* p, unsigned size) {
+    assert(size == 1 && "unexpected status field width");
+
+    if ((void*)env < p && (p - (void*)env) < offsetof(CPUArchState, eip)) {
```

Comment:
Arithmetic on void pointers is undefined behavior.
https://stackoverflow.com/questions/3523145/pointer-arithmetic-for-void-pointer-in-c#3524270

Reply:
I wonder why the compiler didn't catch it. Perhaps we need to add the undefined behavior sanitized flag to clang.

Reply:
Yes, char* cast would be more portable, clang seems in order to keep compatibility of gcc, allow this.